js/scripts.js

```diff
@@ -1,18 +1,14 @@
-function codeSample(title, image, description) {
-  this.title = title;
-  this.image = image;
-  this.description = description;
-}
-
-function project(title, image, link, description) {
-  this.title = title;
-  this.image = image;
-  this.link = link;
-  this.description = description;
-}
+'use strict';
 
 $(document).ready(function() {
-  $('.icon-menu').mouseenter(function (){
-    $('nav ul').display = 'block';
-  });
+  $('.icon-menu').mouseenter(openMenu);
+  $('.menu-close-button').click(closeMenu);
```

Comment:
It is better practice to use the generic listener `.on()` to apply an event handler rather than these specific methods.